### PR TITLE
#2 made basic layout

### PR DESCRIPTION
**Pull request by GustavoAcosta02**
Made basic-ish layout, still needs more tweaks to it though

### CODE REVIEW

**Review by lclemente225 — APPROVED** [review]
Feedback: I noticed that you added CSS to the App.css file. It works for now, however, I see an issue if other people use the same class. It might be useful for debugging if you have CSS folders that pertain to the component that they are styling.

**Comment by Sillor** [comment]
Please pull from `main` first and resolve the merge conflict.

**Review by lclemente225 — APPROVED** [review]
Looks good

File: package-lock.json
Comment (Sillor) [inline package-lock.json:1]: Remove this file

File: src/timetracker/components/login/Login.css
Comment (Sillor) [inline src/timetracker/components/login/Login.css:1]: Most of this can be implemented with tailwind, why not use it?

**Review by Sillor — CHANGES_REQUESTED** [review]

**Comment by AlissonRoss** [comment]
Work on responsiveness for mobile viewport
Other than that, Beautiful work

**Review by AlissonRoss — APPROVED** [review]
image
Wonderful! I am approving this

**Review by Sillor — APPROVED** [review]
Looks good 👍

**Pull request merged**